### ISSUE TITLE
Python binding failed

### ISSUE THREAD

**Issue by manhha1402** [yuecideng/Misc3D#29]
Hi @yuecideng ,
I followed the instruction to install and run examples in the repo and got the error with python binding, for example: `farthest_point_sampling.py`: `TypeError: farthest_point_sampling(): incompatible function arguments. The following argument types are supported`.
I use ubuntu 20.04, open3d installed with CUDA enable. 
Do you know how to solve the issue?.
Thank you

**Comment by yuecideng** [comment]
Hi, how did you build open3d? Did you install open3d python package after build open3d from source?

**Comment by manhha1402** [comment]
Hi, yes. I built Open3d from source and open3d pybind package after building open3d from souce.

**Comment by yuecideng** [comment]
I'm not sure whether the mismatch of open3d data type is caused by the CUDA anbled. Previously I developed misc3d with cpu only version of open3d and did not try to use CUDA enbled open3d with misc3d. I will try it if I have bandwidth for it. 

which function you would like to use in misc3d? If you want to use `farthest_point_sampling`, it has been integrated into open3d since `0.16.0`. @manhha1402

**Comment by manhha1402** [comment]
Hi @yuecideng , 
Actually I want to do the python binding for some C++ functions using Open3d like you but it was't successful. I did the same as you  here: `https://github.com/yuecideng/Misc3D/tree/main/python` and got the same error : `incompatible function arguments. The following argument types are supported`.
Do you know how to fix it?. Thank you

**Comment by yuecideng** [comment]
You may try to check both the misc3d pybind so and open3d pybind so link to the same `libOpen3D.so`.

**Comment by manhha1402** [comment]
Thank you for your suggestion. Do you have a plan develop CUDA implementation for ppf pose estimation or integrate to Open3D?

**Comment by yuecideng** [comment]
Yeah I am doing this and glad to contribute it to Open3D. But it may take some time due to my limited bandwidth.
